### ISSUE TITLE
Redsoft website

### ISSUE THREAD

**Issue by AlexandraDikusar** [redsoft-pro/official-site#2]
5. !!! Очень хорошая фишка - вращающиеся фигуры. Они кликабельны - но пользователь никогда об этом не догадается. Предложение: когда подносишь мышку на стартовой странице к данным фигурам, пусть появляется точка (увеличивается, при перемещении мышки - перемещается за ней).

6. Может как-то упростить данную функцию. Когда очень много раз нажать по одной фигуре - сайт подвисает, фигуры не крутятся

**Comment by AlexOreshkevich** [comment]
Конечно, пользователю надо подсказать как-то, что есть возможность "поиграться". Например, увеличивать узлы дерева при наведении мышкой на них.

Максимальное число кликов, которые обрабатываются, конечно, надо ограничить. Например, если свыше 20 кликов, то не реагировать.
